stubs/CoreGenericFunctions.phpstub

```diff
@@ -1116,7 +1116,13 @@ function preg_quote(string $str, ?string $delimiter = null) : string {}
 /**
  * @psalm-pure
  *
- * @param string|int|float $values
+ * @template TFormat of string
```

Comment:
FYI, You're not forced to use templates for a simple case like this.

Psalm would have understood 
```
 * @return ($format is non-empty-string
 *      ? ($values is non-empty-string|int|float ? non-empty-string : string)
 *      : string)
```
as well

Reply:
Thanks!